seq2seq/nmt_without_attention_v2.py

```diff
@@ -0,0 +1,243 @@
+#!/usr/bin/env python
+
+import sys
+import gzip
+import sqlite3
+import paddle.v2 as paddle
+
+### Parameters
+word_vector_dim = 620
+latent_chain_dim = 1000
+
+beam_size = 3
+max_length = 50
+
+
+def seq2seq_net(source_dict_dim, target_dict_dim, generating=False):
+    decoder_size = encoder_size = latent_chain_dim
+
+    ### Encoder
+    src_word_id = paddle.layer.data(
+        name='source_language_word',
+        type=paddle.data_type.integer_value_sequence(source_dict_dim))
+
+    src_embedding = paddle.layer.embedding(
+        input=src_word_id,
+        size=word_vector_dim,
+        param_attr=paddle.attr.ParamAttr(name='_source_language_embedding'))
+
+    encoder_forward = paddle.networks.simple_gru(
+        input=src_embedding,
+        act=paddle.activation.Tanh(),
+        gate_act=paddle.activation.Sigmoid(),
+        size=encoder_size,
+        reverse=False)
+    encoder_backward = paddle.networks.simple_gru(
+        input=src_embedding,
+        act=paddle.activation.Tanh(),
+        gate_act=paddle.activation.Sigmoid(),
+        size=encoder_size,
+        reverse=True)
+    encoded_vector = paddle.layer.concat(
+        input=[encoder_forward, encoder_backward])
+
```

Comment:
Thanks for your reminder. I will rewrite this part by `bidirectional_gru` and clean up other parts